packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -205,10 +156,22 @@ class _TimePickerHeader extends StatelessWidget {
     onChanged(selectedTime.replacing(hour: newHour));
   }
 
+  TextStyle _getBaseHeaderStyle(TextTheme headerTextTheme) {
+    assert(orientation != null);
+    switch (orientation) {
+      case Orientation.portrait:
+        return headerTextTheme.display3.copyWith(fontSize: 60.0);
```

Comment:
are these numbers from the spec? you should add a comment justifying these numbers.